src/components/common/animations/Success/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Lottie from 'lottie-react-native'
+import { getAnimationData } from '../../../../lib/utils/lottie'
+const { animationData, imageAssetsFolder } = getAnimationData('Success', require('./data'))
+
+class Success extends React.Component {
+  render() {
+    return (
+      <Lottie
+        imageAssetsFolder={imageAssetsFolder}
```

Comment:
i see there always are some common props you pass to the <Lottie>
Probably we could create some pre-configuired Lottie wrapper where those props are pre-set via defaultProps